FeatureSuggestions/kishankunal.md

```diff
@@ -1,25 +1,27 @@
-=======================================================
-        Report Card Management System
-======================================================
-
-
-1 Functions
+# Functions
       A. It should contain student information like his name , roll no, registration number and his marks of different subject of mid term , end term, sesional and of different class test if its considered for creating final weightage.
+      
       B. Flexible report card models - It should have different TEMPLATES or models to get different styled report card .
+      
       C. HOD/HOS should be given the right to create/delete exams as well as they can reshedule the exams also.
+      
       D. Only techer is allowed to enter the marks of student, however HOD/HOs can rectify it if any student wants to re-check.
+      
       E. Review/complaint system - student can review their performace and put a complaint if he wants any rectificaton.
+      
       F. Result Display - Every student would be able to view the marks/grade of students of his own section.
+      
       G. student should be able to view his marks subject wise. Also he can compare his marks with the top scorer.
+      
       H. Teacher can give remarks to student based on their performance which should be private to the students only show  that he could further improve.
+      
       I. Attandance of each subject should be stored ans agreegate should also be displayed.
-      j. teacher would have access to view the attandance of every student in their subject and can give remarks in their      subject.
+      
+      j. teacher would have access to view the attandance of every student in their subject and can give remarks in their subject.
+      
       k. For every subject the basic contact details of teacher could be stored.
 
 
-2. Technology used as suggested
-
-3. Also, we can include a check to get parent consent and remark on his performance on every subject.
+# Also, we can include a check to get parent consent and remark on his performance on every subject.
```

Comment:
the whole line should not act as a title. Put the title in a few words and put the line under that.